src/main/java/nextstep/ladder/Ladder.java

```diff
@@ -0,0 +1,32 @@
+package nextstep.ladder;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class Ladder {
+    private List ladder = new ArrayList<>();
```

Comment:
Java 제네릭을 사용하면 어떨까요? 아래의 글이 도움 되면 좋겠어요.
https://madplay.github.io/post/dont-use-raw-types

```suggestion
    private List<Line> ladder = new ArrayList<>();
```